src/main/java/com/adobe/campaign/tests/integro/phased/PhasedTestManager.java

```diff
@@ -555,12 +559,9 @@ protected static Properties importPhaseData() {
 
             int lt_nrAfterPhase = l_methodMapping.totalClassMethods - lt_nrBeforePhase;
 
-            StringBuilder lt_sb = new StringBuilder(STD_PHASED_GROUP_PREFIX);
-            lt_sb.append(lt_nrBeforePhase);
-            lt_sb.append("_");
-            lt_sb.append(lt_nrAfterPhase);
-
-            l_objectArrayPhased[rows][0] = lt_sb.toString();
+            l_objectArrayPhased[rows][0] = STD_PHASED_GROUP_PREFIX + lt_nrBeforePhase
+                + "_"
+                + lt_nrAfterPhase;
```

Comment:
@krmahadevan Coulld you explain the reason for this change?

Reply:
@baubakg - Best use case for using StringBuilder is when you are building a string incrementally in a for loop. That's not the case here. So there's no reason why we need to be using a StringBuilder. You can see these sort of suggestions from IntelliJ as well.